### ISSUE TITLE
Some questions about training model

### ISSUE THREAD

**Issue by guoaoo** [CorentinJ/Real-Time-Voice-Cloning#862]
I have trained a synthesizer used by default hparams from scratch in 14 days, and there are some plot in Tensorboard.
https://github.com/CorentinJ/Real-Time-Voice-Cloning/blob/7432046efc23cabf176f9fdc8d2fd67020059478/synthesizer/hparams.py#L52-L57
![image](https://user-images.githubusercontent.com/34093221/136230583-44ff7c2a-9f58-4d6b-9974-f226bfa93f1a.png)
![image](https://user-images.githubusercontent.com/34093221/136230521-36bc5fde-1413-49d4-b302-6fd80aea33e6.png)

Some questions to ask:
1. The pretrain model is trained to 295k, is that because it is good enough? The size of the `.pt` file does not increase with the number of training steps.
2. I used the default hparams during training and only used about 6G of GPU memory, there is still a lot left for 12G 2080ti and 32G V100, how do I adjust the hparams to speed up the training? If increase `batch_size`, how much should I increase the `learning rate`? Is `reduction factor` an adjustable number, and will it have a bad effect on the results?
3. After reading #449 , do you recommend that I modify the hparams and train a synthesizer from scratch or just fine-tuning on pretained model?
